01_network_sysapi/multithread/threadSafeQueue.hpp

```diff
@@ -0,0 +1,93 @@
+#ifndef THREADSAFEQUEUE_HPP
+#define THREADSAFEQUEUE_HPP
+
+#include <queue>
+#include <utility>
+
+#include <thread>
```

Comment:
Task requirement violated: **For threading, networking should use platform-dependent API.**